SharkCage/CageManager/CageManager.cpp

```diff
@@ -17,13 +18,22 @@
 #pragma comment(lib, "Rpcrt4.lib")
 
 NetworkManager network_manager(ContextType::MANAGER);
+std::optional<std::wstring> generateUuid();;
```

Comment:
minor nitpick: redundant semicolon, would appreciate it if you could remove it 0:-)